src/node.cc

```diff
@@ -515,6 +534,18 @@ inline void PlatformInit() {
   RegisterSignalHandler(SIGINT, SignalExit, true);
   RegisterSignalHandler(SIGTERM, SignalExit, true);
 
+  {
+    struct sigaction sa;
```

Comment:
Is the current signature of `RegisterSignalHandler` adequate? It only takes `sa_handler`, but not  `sa_sigaction`. If we allow the user overrides this, they probably need the `sa_sigaction` variant as well.